graphql/schema.graphql

```diff
@@ -495,6 +498,30 @@ type SiteBanner {
   theme: String!
 }
 
+type Host{
+  id: String!
+  hostURL: String!
+  distro: DistroInfo
+	startedBy: String!
```

Comment:
I think there's some mixup of tabs and spaces here

Reply:
I'm not really sure why this happens. It's looks normal in VSCode. I'll fix it.